Frontend/CO.CDP.OrganisationApp.Tests/Pages/PersonEntityFactory.cs

```diff
@@ -0,0 +1,66 @@
+using CO.CDP.Person.WebApiClient;
+
+namespace CO.CDP.OrganisationApp.Tests.Pages;
+public static class PersonEntityFactory
```

Comment:
Probably needs a different name. Methods have not much to do with the Person. `ProblemDetailsFactory`? `ErrorFactory`?

Reply:
yes sure. changing it now.